saws-iam/src/main/scala/com/ambiata/saws/iam/Policy.scala

```diff
@@ -1,16 +1,42 @@
 package com.ambiata.saws
 package iam
 
-trait Policy {
+sealed trait Policy {
   def name: String
 }
 
 /** IAM policy. */
 case class InlinePolicy(name: String, document: String) extends Policy
 
+/**
+  * Represents a policy that is managed either by AWS (predfinined) or customer created (shared)
+  *
+  * @param name
+  */
+case class AwsManagedPolicy(val name: String, val arnPrefix: String = "arn:aws:iam::aws:policy") extends Policy {
+  def arn: String = s"$arnPrefix/$name"
```

Comment:
You'll probably struggle to access this, as the type Policy doesn't have it.

I would probably make ArnPrefix its own case class, and have a function which takes an ArnPrefix and a Name and returns an Arn.